source/_docs/configuration/templating.markdown

```diff
@@ -1537,6 +1537,44 @@ When a command template renders to a valid `bytes` literal, then MQTT will publi
 - Template {% raw %}`{{ 16 }}`{% endraw %} renders to payload encoded string `"16"`.
 - Template {% raw %}`{{ pack(0x10, ">B") }}`{% endraw %} renders to a raw 1 byte payload `0x10`.
 
+### Determining types
+
+When working with templates, it can be useful to determine the type of
+returned value from a method or the type of a variable at times.
+
+For this, Home Assistant provides the `typeof()` template function and filter,
+is inspired by the [JavaScript](https://en.wikipedia.org/wiki/JavaScript)
+`typeof` operator. It reveals the type of the given value.
+
+This is mostly useful when you are debugging or playing with templates in
+the developer tools of Home Assistant. However, it might be useful in some
+other cases as well.
```

Comment:
```suggestion
When working with templates, it can be useful to determine the type of
the returned value from a method or the type of a variable at times.

For this, Home Assistant provides the `typeof()` template function and filter,
which is inspired by the [JavaScript](https://en.wikipedia.org/wiki/JavaScript)
`typeof` operator. It reveals the type of the given value.

This is mostly useful when you are debugging or playing with templates in
the developer tools of Home Assistant. However, it might be useful in some
other cases as well.
```